tripal_chado/config/install/tripal.tripalfield_collection.general_chado.yml

```
@@ -1514,6 +1514,33 @@ fields:
               region: content
               weight: 12
 
+    -   name: 'pub_publication_type'
+        content_type: 'pub'
+        label: 'Additional Publication Type'
+        type: 'chado_property_type_default'
+        description: 'Additional descriptors indicating the type of publication'
```

What about something like this to make it more clear.
```suggestion
        description: 'Additional terms further describing the type of publication. This should not include the primary term which is indicated in the "Publication Type" field.'
```

---

Nice improvement